### ISSUE TITLE
Train and Validation Split

### ISSUE THREAD

**Issue by stevenwalton** [microsoft/StyleSwin#17]
What was the train and validation split used? I'm using the checkpoint provided and testing with a validation set of the top 10k, similar to [Co-Mod-GAN](https://github.com/zsyzzsoft/co-mod-gan)'s split (section 5.1). Using this split I am getting a FID of 4.26.

**Comment by ForeverFancy** [comment]
Hi, as written in the paper, for FFHQ and LSUN Church datasets, we randomly sample 50,000 images from the original datasets as validation sets and calculate FID between the validation sets and 50,000 generated images. For CelebA-HQ, we calculated the FID between 30,000 generated images and all the training samples.

**Comment by stevenwalton** [comment]
Thanks. Do you have any uncertainty value for these numbers? Or were all your experiments with the same random seed?

**Comment by ForeverFancy** [comment]
Sorry, we do not uncertainty value of the reported numbers. The random seeds are all set to the default values.

**Comment by stevenwalton** [comment]
Thanks
